protocol/testing/testnet/testnet.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+set -eo pipefail
+
+# This file initializes the home directories of validators / full-nodes for validators run for
+# the private test-net on AWS.
+
+source "./vars.sh"
+
+CHAIN_ID="dydx-testnet-1"
+
+# Define the mapping from version to URL
+declare -A version_to_url
+version_to_url["v5.1.0"]="https://github.com/dydxprotocol/v4-chain/releases/download/protocol%2Fv5.1.1/dydxprotocold-v5.1.1-linux-amd64.tar.gz"
```

Comment:
Should we add `v5.2.0` here?

Reply:
Yeah we should specify previous version. This is getting backported to v5.2.0 branch first though. Will follow up with a PR to set everything for v6.0.0